code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -6,59 +6,90 @@
 	icon_state = "cutout_basic"
 	w_class = 4
 	burn_state = FLAMMABLE
-	var/list/possible_appearances = list("Assistant", "Clown", "Mime", "Traitor", "Nuke Op", "Cultist", "Clockwork Cultist", "Revolutionary", "Wizard", "Shadowling", "Xenomorph", "Swarmer", \
-	"Ash Walker", "Deathsquad Officer", "Ian") //Possible restyles for the cutout; add an entry in change_appearance() if you add to here
+	// Possible restyles for the cutout;
+	// add an entry in change_appearance() if you add to here
+	var/list/possible_appearances = list("Assistant", "Clown", "Mime",
+		"Traitor", "Nuke Op", "Cultist", "Clockwork Cultist",
+		"Revolutionary", "Wizard", "Shadowling", "Xenomorph", "Swarmer",
+		"Ash Walker", "Deathsquad Officer", "Ian", "slaughter demon",
```

Comment:
Might want to capitalize the two new entries or lowercase the others, for consistency